sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingQueryListenerSuite.scala

```diff
@@ -259,15 +260,37 @@ class StreamingQueryListenerSuite extends StreamTest with BeforeAndAfter {
     listenerBus.listeners.toArray.map(_.asInstanceOf[StreamingQueryListener])
   }
 
+  private val testQueryStatus: StreamingQueryStatus = {
+    StreamingQueryStatus(
+      "name", 1, 123, 1.0, 2.0, Some(345),
```

Comment:
Removed testQueryStatus. Rather used StreamingQueryStatus.testStatus which is needed for python testing anyways.